package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@manuscripts/pressroom-js",
   "description": "A web service for manuscript conversion",
-  "version": "4.0.14-LEAN-3321-0",
+  "version": "4.0.14-LEAN-3321-1",
```

Comment:
can you please update the version without using LEAN-3321  since all what is needed here is to update your transform version 2.1.9 and merged to master, nothing needed to be tested for LEAN-3321